src/test/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/TransactionExpirationQueueConsumerTests.kt

```diff
@@ -145,6 +146,40 @@ class TransactionExpirationQueueConsumerTests {
     verify(checkpointer, Mockito.times(1)).success()
   }
 
+  @Test
+  fun `messageReceiver receives legacy activated messages successfully`() {
```

Comment:
Could you add a test against legacy transaction expired event too?

Reply:
Fixed in 342d9d07c4afe10f5223847612fca66c26476bc6